src/oom/oomd-util.c

```diff
@@ -134,13 +134,13 @@ bool oomd_memory_reclaim(Hashmap *h) {
         return pgscan_of > last_pgscan_of;
 }
 
-bool oomd_swap_free_below(const OomdSystemContext *ctx, uint64_t threshold_percent) {
+bool oomd_swap_free_below(const OomdSystemContext *ctx, int threshold_permyriad) {
         uint64_t swap_threshold;
 
         assert(ctx);
-        assert(threshold_percent <= 100);
+        assert(threshold_permyriad <= 10000);
```

Comment:
@poettering Shouldn't we check (in the assert) that `threshold_permyriad` is `>= 0` either?